config/initializers/devise.rb

```diff
@@ -96,7 +96,7 @@
   # a value less than 10 in other environments. Note that, for bcrypt (the default
   # encryptor), the cost increases exponentially with the number of stretches (e.g.
   # a value of 20 is already extremely slow: approx. 60 seconds for 1 calculation).
-  config.stretches = Rails.env.test? ? 1 : 12
+  config.stretches = Rails.env.test? ? 1 : 100_000
```

Comment:
Based on your benchmarks, it looks like 100,000 iterations is about twice as fast as bcrypt's 12 stretches setting (our previous Devise setting). Since we can't use bcrypt, would it make sense to match the slowness of bcrypt by increasing the iterations? I would vote for matching at least a setting of 11, which is Devise's default for bcrypt. I believe 100,000 iterations should be as slow as 11 stretches. Can you confirm that it isn't faster, please?